backend/src/utils/consolidateQueryResult.ts

```diff
@@ -71,7 +70,7 @@ export function consolidateQueryResult(
         nodeIds.add(edge.from);
         subsidiary.push({
           id: edge.from,
-          types: [], // types of subsidiary nodes are unknown, color is yellow either way
+          types: [], // types of subsidiary nodes are unknown, color is special
```

Comment:
is `special` a keyword here for something? If not, write a TODO that this color need to be discussed